forge.config.js

```diff
@@ -45,7 +46,7 @@ module.exports = {
         loadingGif: './assets/loading.gif',
         noMsi: true,
         remoteReleases: '',
-        setupExe: `electron-fiddle-${package.version}-setup-${process.arch}.exe`,
+        setupExe: `electron-fiddle-${version}-setup.exe`,
```

Comment:
Are we not going to distribute setup files for multiple Windows arches, then?

Reply:
We are, I'll just have to handle this manually for now. Is there a way to get whatever architecture `forge` is actually using?

I'll merge this for now, but I'd love to do something like this:

```
import { args } from '@electron-forge/cli'
const { arch } = args
```

Reply:
I think it'd have to be a [`postMake` hook](https://v6.electronforge.io/configuration#postmake) to rename the artifact with the correct arch. I'd verify with Sam but he's :palm_tree: :smile:

Reply:
I only muted `electron/electron` 😄 I'd go crazy if I didn't use GH at all for 5 weeks 😆 

You're looking for this hack 😄 Ignore the comment, I'm probably gonna just make it official API 👍 

https://github.com/electron-userland/electron-forge/blob/master/packages/maker/base/src/Maker.ts#L60-L68

Basically, your `config` property can be function instead that takes in the target arch and returns the config object

Reply:
I'll probably end up doing something similar to the `fromBuildIdentifier` helper --> https://v6.electronforge.io/configuration#build-identifier